speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MsgpackDeserializeTests.cpp

```diff
@@ -14,20 +14,20 @@ TEST(MsgpackDeserializeTests, Deserialize)
     float controlSteerValue = random.NextFloat(1000);
     float controlBrakeValue = random.NextFloat(1000);
 
-    std::vector<std::string> mockMessageFromBlackBox = {std::to_string(controlSteerValue), std::to_string(controlBrakeValue)};
+    std::vector<std::string> mockMessageFromBlackBox = { std::to_string(controlSteerValue), std::to_string(controlBrakeValue) };
     msgpack::sbuffer sbuffer;
     msgpack::pack(sbuffer, mockMessageFromBlackBox);
 
     SocketBlackBox<DriveSituationMock> socketBlackBox;
     socketBlackBox.Initialize();
 
-    socketBlackBox.m_variablesToReceive = {"Steer", "Brake"};
+    socketBlackBox.m_variablesToReceive = { "Steer", "Brake" };
```

Comment:
Make all string literals in this file #defines or constants. I see that you use them at multiple places, and if you ever want to edit one you'd need to edit all the string literals one-by-one hoping that you don't forget any. By making it a variable you can edit all at once. This is applies for "NON_EXISTING_DECISION_KEY" etc.
```suggestion
    socketBlackBox.m_variablesToReceive = { STEER, BRAKE };
```